test/configuration_file_test.go

```diff
@@ -0,0 +1,115 @@
+package test
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/santhosh-tekuri/jsonschema/v5"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+)
+
+func Test_validateTestConfigurationFiles(t *testing.T) {
+	err := validateTestConfigurationFiles("../jsonschema/golangci.next.jsonschema.json", ".")
```

Comment:
does it depend on directory where `go test` was executed?

I remember 
```go
 _, filename, _, _ := runtime.Caller(0)
```
pattern for tests using related paths

Reply:
It depends on the test file position.